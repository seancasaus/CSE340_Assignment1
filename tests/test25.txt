foAxj (((s)|(J))|((z).(R))).(J) , g7mJO (2).((l)*) , L6 q , pY0vD s , tIykT (q).(((o)|(j))|(v)) #
" s2q2ll ssqqjsJ2l2llqv2 sqjqj2lqosJzRJ2 " 
