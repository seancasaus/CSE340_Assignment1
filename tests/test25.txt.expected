pY0vD , "s"
g7mJO , "2"
L6 , "q"
g7mJO , "2ll"
pY0vD , "s"
pY0vD , "s"
L6 , "q"
tIykT , "qj"
foAxj , "sJ"
g7mJO , "2l"
g7mJO , "2ll"
tIykT , "qv"
g7mJO , "2"
pY0vD , "s"
tIykT , "qj"
tIykT , "qj"
g7mJO , "2l"
tIykT , "qo"
foAxj , "sJ"
foAxj , "zRJ"
g7mJO , "2"
